content/maxwell3_fdem/sphere_freespace.rst

```diff
@@ -0,0 +1,262 @@
+.. _sphere_freespace:
+
+Response from a Conducting and Permeable Sphere in Free-Space
+-------------------------------------------------------------
+
+Introduction
+============
+
+Frequency-domain responses from ore bodies and unexploded ordnance items are of great practical importance to geophysicists.
+On a fundamental level, these responses can be understood by considering the excitation of a conductive and magnetically permeable sphere, under the influence of a harmonic inducing field.
+Here, expressions derived by Wait (1951) are used to characterize the excitation of the sphere in terms of an induced dipole moment.
+We examine how a target's dipole moment depends on its size and physical properties.
+A simple model for predicing the sphere's dipole response in free-space is then presented.
+For a sufficiently compact target, the free-space response represents a low-order approximation to the response of a conductive and permeable object within a resistive medium.
+
+
+Plane Wave Excitation of a Conducting and Magnetically Permeable Sphere
+=======================================================================
+
+In this section, equations derived by Wait (1951) are used to describe the induced magnetic dipole moment of a conductive and permeable sphere, resulting from excitation by a uniform magnetic field (Figure ??).
```

Comment:
Replace Figure ?? with correct notation:
example: 
if figure is: 
 .. figure:: images/Ienc.png
    :align: right
    :scale: 20% 
    :name: Ienc

in-text reference is then:
(See :numref:`Ienc`)